ctk/common/src/main/kotlin/org/codice/compliance/verification/core/CoreVerifier.kt

```diff
@@ -45,12 +45,6 @@ abstract class CoreVerifier(protected val node: Node) {
                                   expectedStatusCode: String) {
             SchemaValidator.validateSAMLMessage(node)
             val status = node.children("Status")
-            if (status.size != 1)
```

Comment:
❓ Was this taken out because you can have more than 1 status, or because Schema Validation will catch it?

Reply:
Schema validation will catch it.